### PR TITLE
Revert "PEF: Implement "daily" sessions."

### PR DESCRIPTION
**Pull request by jediandrey**
This reverts commit ec101de33e541a99f431587971cd6a1286d65ee8.

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature, please create PR against 8.x-1.xx-dev branch.

If this is a bug fix - use 8.x-1.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build861 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build861 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build861 |
| Installation process | http://install.openy.cibox.tools/build861/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/861/

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 1]
```diff
@@ -400,17 +392,20 @@ public static function loadSessionSchedule(NodeInterface $node) {
...
     $dates = $node->field_session_time->referencedEntities();
     foreach ($dates as $date) {
-      if (empty($date) || empty($date->field_session_time_date->getValue())) {
+      if (empty($date) || empty($date->field_session_time_days->getValue()) || empty($date->field_session_time_date->getValue())) {
         continue;
       }
-
       $schedule_item = [
-        'frequency' => $date->field_session_time_frequency->getValue()[0]['value'],
+        'frequency' => 'weekly',
```
Comment (diffyDemoAccess) [inline modules/custom/openy_repeat/src/RepeatManager.php]: we can safely remove 'frequency' completely

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 2]
```diff
@@ -229,52 +229,32 @@ public static function calcSessionInstancesBySchedule(array $session_schedule, $
...
-        }
-        $endDate = new \DateTime($schedule_item['period']['to']);
-        while (strtolower($endDate->format('l')) !== $weekDay) {
-          $endDate->modify('-1 day');
-        }
-
-        $start = $startDate->format('Y-m-d') . 'T' . $schedule_item['time']['from'];
-        $end = $endDate->format('Y-m-d') . 'T' . $schedule_item['time']['to'];
-        $dates = [[
+      if ($schedule_item['frequency'] == 'weekly' || $schedule_item['frequency'] == 'daily') {
```
Comment (ygerasimov) [inline modules/custom/openy_repeat/src/RepeatManager.php]: please remove 'frequency' completely

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 3]
```diff
@@ -346,7 +327,12 @@ public static function combineDates($origin_dates, $exclusions) {
       if ($skipInstance && !$prevSkipInstance) {
         $period = array_pop($resultingPeriods);
         $close = clone $start;
-        $close->modify('-1 week');
+        if ($schedule_item['frequency'] == 'weekly') {
```
Comment (ygerasimov) [inline modules/custom/openy_repeat/src/RepeatManager.php]: please remove 'frequency' completely

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 4]
```diff
@@ -357,7 +343,13 @@ public static function combineDates($origin_dates, $exclusions) {
         $period = ['from' => clone $start];
         array_push($resultingPeriods, $period);
       }
-      $start->modify('+1 week');
+
+      if ($schedule_item['frequency'] == 'weekly') {
```
Comment (ygerasimov) [inline modules/custom/openy_repeat/src/RepeatManager.php]: please remove 'frequency' completely

File: modules/custom/openy_repeat/src/RepeatManager.php (hunk 1 above)
Comment (ygerasimov) [inline modules/custom/openy_repeat/src/RepeatManager.php]: please remove 'frequency' completely

**Comment by bjwilkins** [comment]
!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/899/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1260 of commit 69450a39b33eb8d21b1c4e00be43981330b61771 automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build899/openy-profile # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6535401635816884239.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
Could not remove the credential section from the git configuration
Build was aborted
Aborted by anonymous
Slack Notifications is waiting for a checkpoint on PR_BUILDER_COMPOSER #898
Couldn't read comment file
java.io.FileNotFoundException: File '/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build899/commentinfo.md' does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1748)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:124)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:201)
	at hudson.model.Run.execute(Run.java:1786)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
```

**Pull request closed without merging**